observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,72 @@ These release notes track the development of the Network Observability Operator
+and see xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-quickfilternw-observe-network-traffic[Quick filters].
+
+
+[id="network-observability-ebpf-collection-filtering-1.6"]
+==== eBPF collection rule-based filtering
+You can use rule-based filtering to reduce the volume of created flows. When this option is enabled, the *Netobserv / Health* dashboard for eBPF agent statistics has the *Filtered flows rate* view. For more information, see xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-ebpf-flow-rule-filter_nw-observe-network-traffic[eBPF flow rule filter].
+
+[id="network-observability-technology-preview-1.6"]
+=== Technology preview features
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use. Note the following scope of support on the Red Hat Customer Portal for these features:
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+[id="network-observability-cli-1.6"]
+==== Network Observability CLI
+The Network Observability CLI allows you to debug and troubleshoot network traffic issues without needing to install the Network Observability Operator. You can capture and visualize flow and packet data in real-time with no persistent storage requirement during the capture. For more information, see 
+// commenting until CLI feature doc is merged 
+// xref:../../observability/network_observability/netobserv_cli/netobserv-cli-install.adoc#network-observability-netoberv-cli-about_netobserv-cli-install[Network Observability CLI]
+
+[id="network-observability-operator-1.6.0-bug-fixes"]
+=== Bug fixes
+* Previously, a dead link to the OpenShift documentation was displayed in the OLM form for `FlowMetrics` API creation. Now the link has been updated to point to a valid page. (link:https://issues.redhat.com/browse/NETOBSERV-1607[*NETOBSERV-1607*])
+* Previously, the Operator description in Operator Hub was displaying a broken link to the documentation. With this fix, this link is restored. (link:https://issues.redhat.com/browse/NETOBSERV-1544[*NETOBSERV-1544*])
+* Previously, if Loki was disabled and the Loki `Mode` was set to `LokiStack`, or if Loki manual TLS configuration was configured, the Operator still tried to read the Loki CA certificates. With this fix, when Loki is disabled, the Loki certificates are not read, even if there are settings in the Loki configuration. (link:https://issues.redhat.com/browse/NETOBSERV-1647[*NETOBSERV-1647*])
+* Previously, the must-gather plugin for the Network Observability Operator was only working on `amd64` architectures and failing on all others because the plugin was using amd64 for `oc` binary. Now, the Network Observability Operator must-gather plugin collects logs on any architecture platform. (link:https://issues.redhat.com/browse/NETOBSERV-1618[*NETOBSERV-1618*])
```

Comment:
```suggestion
* Previously, the `oc` `must-gather` plugin for the Network Observability Operator was only working on `amd64` architectures and failing on all others because the plugin was using on the `amd64` architecture for the `oc` binary. Now, the Network Observability Operator must-gather plugin collects logs on any architecture platform. (link:https://issues.redhat.com/browse/NETOBSERV-1618[*NETOBSERV-1618*])
```
You can use the first-time `oc` admonition for the first one if you use the common attributes file.